src/components/shared/toggleButton/toggleButton.js

```diff
@@ -1,15 +1,26 @@
 import React from "react";
+import PropTypes from "prop-types";
 import "./toggleButton.scss";
 
-const TogleButton = () => {
+const ToggleButton = ({ children }) => {
```

Comment:
Does it works as expected? You made right changes, where you passed className into this component(in HeaderRightBlock), but this prop didn't handle inside the component

Reply:
It was not working, now I see I should handle it in props. But in the end, after refactoring, this class changes only one small margine, thats why I had not seen its not workin. I will drop not nessesary class and pass the margine to ToggelButton. Thank you.